azure-iot-device/tests/common/pipeline/fixtures.py

```diff
@@ -23,8 +23,51 @@ def fake_base_exception():
     return helpers.UnhandledException()
 
 
+class MockEvent(pipeline_events_base.PipelineEvent):
```

Comment:
Is this really a `mock`? It seems like it's just a subclass of an abstract class, for test purposes, with none of the expected functionality of a Mock() or MagicMock()